src/compile-middleware.ts

```diff
@@ -75,35 +71,33 @@ export const babelCompileCache = LRU<string>(<LRU.Options<string>>{
 
 
 export function babelCompile(forceCompile: boolean): RequestHandler {
-  if (forceCompile == null) {
-    forceCompile = false;
-  }
-
   return transformResponse({
-    shouldTransform(_request: Request, response: Response) {
-      return isSuccessful(response) &&
-          compileMimeTypes.indexOf(getContentType(response)) >= 0;
+    shouldTransform(request: Request, response: Response) {
+      // We must never compile the Custom Elements ES5 Adapter or other
+      // polyfills/shims.
+      const isPolyfill = url.parse(request.url)
+                             .pathname.split('/')
+                             .indexOf('webcomponentsjs') >= 0;
```

Comment:
You can use `Array#includes` instead of `indexof`. Also, rather than split the path then check the results, you could just do `pathname.includes('webcomponentsjs/')`